compiler/semexprs.nim

```diff
@@ -505,13 +505,15 @@ proc semOpAux(c: PContext, n: PNode) =
       a.typ = a[1].typ
     else:
       n[i] = semExprWithType(c, a, flags)
-
 proc overloadedCallOpr(c: PContext, n: PNode): PNode =
   # quick check if there is *any* () operator overloaded:
   var par = getIdent(c.cache, "()")
   var amb = false
   if searchInScopes(c, par, amb) == nil:
     result = nil
+  elif n.hasSubnodeWith(nkClosedSymChoice):
```

Comment:
two situations i found: overloaded generic procs result in nkCall(nkClosedSymChoice), overloaded generic templates/macros result in nkCall(nkBracketExpr(nkClosedSymChoice)).
i don't think opensymchoice is possible.
i'm hoping here i'm not casting too wide a net, say if a sym had a closedsymchoice in its params? i couldn't figure out how to write that breaking test.